src/UI/ReplyActions.jsx

```diff
@@ -18,6 +18,9 @@ class ReplyActions extends Component {
 	}
 
 	handleSubmit = () => {
+		if(!this.state.reply)
```

Comment:
Hou er rekening mee dat meerdere dingen true kunnen terug geven. Weet niet of je die afvangt?
https://developer.mozilla.org/en-US/docs/Glossary/Truthy